src/NuGet.Core/NuGet.PackageManagement/Telemetry/ActionsTelemetryEvent.cs

```diff
@@ -26,27 +26,6 @@ public class ActionsTelemetryEvent : ActionEventBase
 
         public const string NugetActionEventName = "NugetAction";
 
-        public NuGetOperationType OperationType => (NuGetOperationType)base[nameof(OperationType)];
-    }
-
-    /// <summary>
-    /// Define nuget operation type values.
-    /// </summary>
-    public enum NuGetOperationType
-    {
-        /// <summary>
-        /// Install package action.
-        /// </summary>
-        Install = 0,
-
-        /// <summary>
-        /// Update package action.
-        /// </summary>
-        Update = 1,
-
-        /// <summary>
-        /// Uninstall package action.
-        /// </summary>
-        Uninstall = 2,
+        public NuGetProjectActionType OperationType => (NuGetProjectActionType)base[nameof(OperationType)];
```

Comment:
If I am understanding correctly, `OperationType` is emitted telemetry.  If so, why not keep `NuGetOperationType` and remove `NuGetProjectActionType`.  If the current change --- removing `NuGetOperationType` --- makes more sense, why not update parameter names (`operationType` -> `actionType`).  This property can remain as-is for telemetry.

Reply:
NuGetProjectActionType is used more heavily overall. 80+ references vs 20 of NuGetOperationType. 

NuGetProjectActionType  is also used more in public types. More notable in the NuGetProjectAction which itself is a type that's used a lot.